README.md

```diff
@@ -1,5 +1,20 @@
 # Xamarin.iOS & Xamarin.Mac #
 
+|              | Status                                    |
+|--------------|-------------------------------------------|
+| master       | [![xamarin-macios-builds-master][1]][2]   |
+| xcode8.2     | [![xamarin-macios-builds-xcode8.2][3]][4] |
+| pr-builder   | [![xamarin-macios-pr-builder][5]][6]      |
```

Comment:
Also I don't think it makes much sense to show the results for the xcode8.2 branch in master. OTOH it would be annoying to have to update the links every time we create a new branch, so not sure what's best here.